Core/Src/leds/debug_leds_handler.c

```diff
@@ -27,24 +27,24 @@ void debug_leds(void* argument) {
     for (;;) {
         ECU_ENABLE_BREAKPOINT_DEBUG();
 
-        // espera ate alguma mensagem chegar
+        // wait until some message arrives
         osMessageQueueGet(q_debug_leds_messageHandle, &message, NULL, osWaitForever);
         switch (message.control) {
             case ON:
-                // acende o led escolhido
+                // lights up the chosen led
                 HAL_GPIO_WritePin(GPIOE, message.lednumber, GPIO_PIN_RESET);
```

Comment:
```suggestion
                HAL_GPIO_WritePin(GPIOE, message.led_number, GPIO_PIN_RESET);
```
Mudar nome do enum. Isso foi erro meu que só reparei agr rs, conserta por favor